src/libos/crates/io-uring-callback/src/lib.rs

```diff
@@ -388,6 +392,30 @@ impl IoUring {
         }
     }
 
+    /// Wait new I/O completions in the completions queue of io_uring.
+    ///
+    /// Upon receiving completed I/O, the corresponding user-registered callback functions
+    /// will get invoked and the `IoHandle` (as a `Future`) will become ready.
+    pub fn wait_completions(&self) {
```

Comment:
If `poll_completions` returns the number of completions, then `wait_completions` would be simply equivalent to
```rust
while io_uring.poll_completions() == 0 {}
```

Reply:
``` Rust
/// Return the number of I/O completions.
pub fn poll_completions(&self) -> usize {}

/// Wait for at least min_complete new I/O completions in the completions queue of io_uring.
/// Return the number of I/O completions.
pub fn wait_completions(&self, min_complete: usize) -> usize {
    // implemented by self.poll_completions()
}
```

Reply:
`busywait_completions`